lib/ansible/modules/monitoring/zabbix/zabbix_action.py

```diff
@@ -64,6 +64,7 @@
         description:
             - Default operation step duration. Must be greater than 60 seconds. Accepts seconds, time unit with suffix and user macro.
         default: '60'
+        required: true
```

Comment:
There are 2 places where `esc_period` is defined in argument_spec, but none of it has `required=True`, so this change would make and inconsistency between module documentation and actual argument definition.

Reply:
This is a valid change as it is specified in [Zabbix API action object documentation](https://www.zabbix.com/documentation/3.2/manual/api/reference/action/object) as well. Please adjust [this line](https://github.com/ansible/ansible/blob/62b3fec7055cfb1eef6b95cace8dd324a16fd97b/lib/ansible/modules/monitoring/zabbix/zabbix_action.py#L1677) to reflect this documentation change - `required=True`.